nautobot/cloud/tests/test_api.py

```diff
@@ -0,0 +1,46 @@
+from nautobot.cloud.models import CloudAccount
+from nautobot.core.testing import APIViewTestCases
+from nautobot.dcim.models import Manufacturer
+from nautobot.extras.models import SecretsGroup
+
+
+class CloudAccountTest(APIViewTestCases.APIViewTestCase):
+    model = CloudAccount
+
+    @classmethod
+    def setUpTestData(cls):
+        secrets_groups = (
+            SecretsGroup.objects.create(name="Secrets Group 1"),
+            SecretsGroup.objects.create(name="Secrets Group 2"),
+            SecretsGroup.objects.create(name="Secrets Group 3"),
+        )
+        manufacturers = Manufacturer.objects.all()
+        cls.create_data = [
+            {
+                "name": "Account 1",
+                "account_number": "1238910123",
+                "provider": manufacturers[0].pk,
+                "secrets_group": secrets_groups[0].pk,
+            },
+            {
+                "name": "Account 2",
+                "account_number": "5123121012",
+                "provider": manufacturers[1].pk,
+                "secrets_group": secrets_groups[1].pk,
+            },
+            {
+                "name": "Account 3",
+                "account_number": "6782109915",
```

Comment:
Whose phone numbers are these? :-)

Reply:
I put my head on the keyboard pretty much

Reply:
:-) Just jumped out at me because 678 is an Atlanta area code.